aggregator/src/main/java/com/redhat/cloud/notifications/config/AggregatorConfig.java

```diff
@@ -34,14 +40,27 @@ private static String toggleName(String feature) {
     @Inject
     Unleash unleash;
 
+    @PostConstruct
+    void postConstruct() {
+        fetchAggregationBasedOnEvents = toggleRegistry.register("fetch-aggregation-based-on-events", true);
+    }
+
     void logConfigAtStartup(@Observes Startup event) {
 
         Map<String, Object> config = new TreeMap<>();
         config.put(UNLEASH, unleashEnabled);
-
+        config.put(fetchAggregationBasedOnEvents, isAggregationBasedOnEventEnabled());
         Log.info("=== Startup configuration ===");
         config.forEach((key, value) -> {
             Log.infof("%s=%s", key, value);
         });
     }
+
+    public boolean isAggregationBasedOnEventEnabled() {
```

Comment:
Here's a question for both `aggregator` and `engine`: should we test the new aggregation way with internal org IDs before making it available to all customers?

Reply:
done, with an additional flag